api/handlers/metadata.go

```diff
@@ -23,46 +23,67 @@ import (
 // GetMetadataHandlerFunc returns metadata of the keptn installation
 func GetMetadataHandlerFunc(params metadata.MetadataParams, principal *models.Principal) middleware.Responder {
 
+	handler := newMetadataHandler()
+
+	return handler.getMetadata()
+}
+
+func newMetadataHandler() metadataHandler {
+	var clientSet kubernetes.Interface
 	logger := keptnutils.NewLogger("", "", "api")
-	logger.Info("API received a GET metadata event")
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		logger.Debug("Could not get InClusterConfig, will skip k8s-deployments.")
```

Comment:
Does it make sense that we log the error?

Reply:
absolutely, I have added the error message to the log output